databricks/koalas/base.py

```diff
@@ -558,7 +558,7 @@ def is_monotonic(self):
 
         Returns
         -------
-        is_monotonic : boolean
+        is_monotonic : bool
```

Comment:
Shall we fix it in `is_monotonic_decreasing` as well?

Reply:
Fixed.